Sources/SwiftUIBackports/Shared/StateObject/StateObject.swift

```diff
@@ -0,0 +1,111 @@
+import Combine
+import SwiftUI
+
+extension Backport where Content == Any {
```

Comment:
One minor thing I'd suggest instead of `Content == Any` is `Content: ObservableObject`; then you can use `Content` in the implementation of `@StateObject` instead of needing to introduce a second generic parameter.

Reply:
Great suggestion, done! → https://github.com/shaps80/SwiftUIBackports/pull/2/commits/7ea58c07ccdda74cb92900a9c4aaedd3846db545

Reply:
> One minor thing I'd suggest instead of `Content == Any` is `Content: ObservableObject`; then you can use `Content` in the implementation of `@StateObject` instead of needing to introduce a second generic parameter.

Oh I missed that, great idea!